sklearn/feature_extraction/text.py

```diff
@@ -876,14 +876,20 @@ class CountVectorizer(_VectorizerMixin, BaseEstimator):
 
     ngram_range : tuple (min_n, max_n), default=(1, 1)
         The lower and upper boundary of the range of n-values for different
+<<<<<<< HEAD
```

Comment:
There seems to be some text added by an uncompleted git merge (`<<<<<<< HEAD`). This should be removed

Reply:
oops! My bad! Will fix and push again!